sim/src/sim/queries.rs

```diff
@@ -371,6 +371,13 @@ impl Sim {
     pub fn infinite_parking(&self) -> bool {
         self.parking.is_infinite()
     }
+
+    pub fn all_waiting_people(&self) -> BTreeMap<PersonID, Duration> {
+        let mut delays = BTreeMap::new();
+        self.walking.all_waiting_people(self.time, &mut delays);
+        self.driving.all_waiting_people(self.time, &mut delays);
```

Comment:
For my own understanding, in this context does `driving` capture bikes/buses too, or are they currently omitted from delay considerations?

Reply:
Here, bikes are included, but buses excluded. Buses have no `PersonID` associated with them. Originally I wanted this mapping to have `AgentID` instead (which would let us show delayed buses), but later I'm joining up with `UnzoomedAgent`s, which don't include `AgentID`. I'm hesitant to add more fields to that struct without perf testing, because it's heavily used (every single draw call, for the agent circles on the minimap or large unzoomed map) and I recall having slimmed down this struct for speedup previously.

Slightly more generally, `DrivingSimState` handles the "low level" mechanics of all vehicles (bikes, buses, cars). As opposed to `WalkingSimState`. Buses additionally have `TransitSimState` as a higher-level manager, roughly analogous to `TripManager` for most people